editoast/src/views/paced_train.rs

```diff
@@ -128,6 +129,43 @@ async fn get_by_id(
     Ok(Json(paced_train))
 }
 
+/// Update a train schedule and its result
```

Comment:
```suggestion
/// Update a paced train and its result
```

what is the "result" referring to?

Reply:
This is the definition of `PacedTrainResult` which *I suppose* is what the "result" refers to here. Can't it just be named `PacedTrain`? It would make the OpenAPI schema clearer as well. Wdyt?

Reply:
It would be nice to have a naming convention, such as *Model/*Row for database structs, just the name for OpenAPI structs, and *Railjson for RailJSON for examples ?

Reply:
My take is that RailJson objects are supposed to represent the objects completely, so I'd strip the *Railjson suffix for those. But if they are augmented with additional information (for a view response, like `PacedTRainResult` here), I'd say that's where we need a suffix. Since `PAcedTrainResult` only exists for view responses, I'd suggest the following names, **especially for the OpenAPI**:

* `PacedTrain` instead of `PacedTrainBase`, since the `Base` suffix doesn't mean anything in the `editoast_schemas` context
* `PacedTrainResponse` instead of `PacedTrainResult`
* We can do the same of `TrainSchedule` definitions

Wdyt? @younesschrifi @flomonster

Reply:
What you are suggesting sound good.

Reply:
If we rename PacedTrainBase to PacedTrain, what should we name the current PacedTrain (model)? PacedTrainModel ? @flomonster @leovalais @younesschrifi

Reply:
We can keep `PacedTrain` too, the namespace is different 🤷

Reply:
> We can keep PacedTrain too, the namespace is different 🤷

Agreed for code, but we still need to tell apart the model and the schema in the OpenAPI. I can see two ways to handle this:

1. Never return the model the endpoints and always convert it into the schema. That works well with the concept of "editoast_schemas". Not sure how well this applies to all our models though.
2. Have a prefix like `Stored-` or `Persisted-` (`Model` doesn't really makes sense in the OpenAPI imo, I see it more as a programming class of objects)

Reply:
I find it quite confusing to use the same name, even if it’s in a different namespace. Personally, I would prefer adding a prefix like ‘Stored’ or ‘Persisted’.

Would you like me to make this change? If so, should I adjust the prefix or the different namespaces?

Reply:
The names are defined in two different namespaces, `models::PacedTrainModels` is redundant imo, whereas `models::PacedTrain` conveys all the information we need. And if we're in a context without ambiguity, we can always `use models::PacedTrain`.

But we need the prefix/suffix in the OpenAPI since all schemas are defined in the same namespace though.

Reply:
Hello @leovalais, I’ve made the changes. Could you please review them?